packet/flatcar-linux/kubernetes/cl/controller.yaml.tmpl

```diff
@@ -6,15 +6,16 @@ systemd:
       dropins:
         - name: 40-etcd-cluster.conf
           contents: |
+            [Unit]
+            Requires=coreos-metadata.service
+            After=coreos-metadata.service
             [Service]
+            EnvironmentFile=/etc/kubernetes/etcd.config
```

Comment:
Isn't `EnvironmentFile=/run/metadata/flatcar` needed for the `ExecStartPre` line?

Reply:
Sorry that script was not needed.